.github/workflows/bootstrap.yml

```diff
@@ -40,17 +40,17 @@ jobs:
       - name: Install GHC
         run: |
           cd $(mktemp -d)
-          curl -sLO "https://downloads.haskell.org/~ghc/8.10.4/ghc-8.10.4-x86_64-apple-darwin.tar.xz"
+          curl -sLO "https://downloads.haskell.org/~ghc/8.10.7/ghc-8.10.7-x86_64-apple-darwin.tar.xz"
```

Comment:
what's this? Why not use ghcup as well? It's also pre-installed on mac.

Reply:
Thanks, I will try that on the master branch version of this PR. This one I will leave in peace.